### DISCUSSION TITLE
Path of Titans

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#38] [Game request]
No info available

**Comment by UpFromTheDepths** [comment]
We could really use Path of Titans support. PoT is meant to be played on a server with a high player count for the best experience, and for a game that will be able to support up to 200 players per server, other server hosts charging for player slots will make the costs insanely expensive. They've recently announced potential console support too (They haven't confirmed which consoles yet though) which will drive the player count in the game even higher considering this game genre has no competition on those platforms

**Reply by Decollates** [reply]
Definitely agree with Depths!

**Comment by Decollates** [comment]
Bumping this. With Path of Titans now releasing for consoles, there's been a significant increase in player activity and server hosting. I believe the effort spent into bringing Path of Titans to GSA would be worth it.

**Comment by Marius9308** [comment]
Maybe reconsider? Here is the guide to setting up a dedicated server. https://github.com/Alderon-Games/pot-community-servers/wiki

**Comment by zerafox** [comment]
I was hoping this had been added or was headed in that direction the other day.
Besides this I can't seem to find any further info on adding Path of Titans support.
